### ISSUE TITLE
NPE when kafka topic doesn't exist

### ISSUE THREAD

**Issue by je-ik** [O2-Czech-Republic/proxima-platform#134]
```
Caused by: java.lang.NullPointerException
	at cz.o2.proxima.direct.kafka.KafkaLogReader.getPartitions(KafkaLogReader.java:142)
	at cz.o2.proxima.beam.direct.io.DirectUnboundedSource.<init>(DirectUnboundedSource.java:125)
	at cz.o2.proxima.beam.direct.io.DirectUnboundedSource.of(DirectUnboundedSource.java:52)
	at cz.o2.proxima.beam.direct.io.DirectDataAccessorWrapper.createStream(DirectDataAccessorWrapper.java:81)
	at cz.o2.proxima.beam.core.BeamDataOperator$StreamDescriptor.createStream(BeamDataOperator.j
```

**Comment by je-ik** [comment]
cc @dmvk

**Comment by je-ik** [comment]
Closed via datadrivencz#288
